helm/scylla-manager/templates/manager_deployment.yaml

```diff
@@ -21,6 +21,7 @@ spec:
       labels:
         {{- include "scylla-manager.labels" . | nindent 8 }}
     spec:
+      serviceAccountName: {{ include "scylla-manager.serviceAccountName" . }}
```

Comment:
I noticed this ServiceAccount is not created by any template. 

You also have to execute `make update` to update the autogenerated files.

Reply:
Yes, noticed that in #467 😅 I've should have updated everything now 👍